### ISSUE TITLE
Public API to access database data

### ISSUE THREAD

**Issue by Illidanz** [vgmdb/tracker#68]
Will probably be based on GraphQL.

**Comment by romanoh** [comment]
Sorry for my noob question, but this will include Json?

**Comment by Illidanz** [comment]
Yes. GraphQL is just the query language but the output is normal JSON.

**Comment by romanoh** [comment]
is is the same way as hufman as made? https://github.com/hufman/vgmdb
i ask because i use that json a lot. Grazie mille. http://vgmdb.info/

**Comment by Illidanz** [comment]
The output will be similar yeah, you should be able to replace it with this new one pretty easily.

**Comment by timlehr** [comment]
Sweet! I was about to commit some time to add some of the missing stuff to the hufman API, but with this news I might wait on the new API instead. Any timeline?

**Comment by Illidanz** [comment]
No ETA yet, sorry.

**Comment by mertvn** [comment]
@Illidanz Any progress on this? I think a lot of people would appreciate it.

**Comment by Illidanz** [comment]
> @Illidanz Any progress on this? I think a lot of people would appreciate it.

There was some progress but it's not ready yet, sorry. The plan is to release a read-only API possibly this year, but it might get delayed.
